src/components/OnBoarding/FirstForm.jsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from 'react';
+import propTypes from 'prop-types';
+import { Box, FormControl, FormLabel, Input, Button, Flex, Text, Image, HStack, SimpleGrid } from '@chakra-ui/react';
+import FPH_LOGO from './fph_logo.png';
+
+const FirstForm = ({ handleChange, nextStep }) => {
+  const [businessName, changeBusinessName] = useState('');
+  const [firstName, changeFirstName] = useState('');
+  const [lastName, changeLastName] = useState('');
+  const [email, changeEmail] = useState('');
+  const [position, changePersonPosition] = useState('');
+  const [website, changeWebsite] = useState('');
```

Comment:
dont create these states, use the states from `formData` in here, such that when we press the back button, we don't lose all of our entries